.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       cmake-args: '-DNUMBER_OF_WORKERS=4 -DLF_THREADED=1'
 
   build-rti:
-    uses: lf-lang/reactor-c/.github/workflows/build-rti.yml@main
+    uses: lf-lang/reactor-c/.github/workflows/build-rti.yml@redo-rti
```

Comment:
I guess this should change back to 
```
uses: lf-lang/reactor-c/.github/workflows/build-rti.yml@main
```
as part of the merge into main?

Reply:
Marten usually does it after merged to main.

Reply:
I just done it as the tests all passed.